Note to change `ViziaPlug` to `<YourPluginName>` in code examples

I followed the instructions in the guide, but I ended up getting this error. I'm new to the language, so it's possible I'm doing something wrong.

My steps, from a new nih-plug cookiecutter project were to add nih vizia to the cargo.toml file and copy the two snippets from the ["adding a basic vizia gui"](https://github.com/vizia/vizia-plug-book/blob/dd9a2e73c54ecf26b42c7c25ca860b500dee32f3/src/getting_started.md#adding-a-basic-vizia-gui) section of the guide to the new `editor.rs` and existing `lib.rs` files in my project. Am I missing a step? I apologize if this is a silly question, it's probably just because I don't understand the language that well.

```
error[E0432]: unresolved import `crate::ViziaPlugParams`
 --> src/editor.rs:6:5
  |
6 | use crate::ViziaPlugParams;
  |     ^^^^^^^^^^^^^^^^^^^^^^ no `ViziaPlugParams` in the root
```

--- a/src/getting_started.md
+++ b/src/getting_started.md
@@ -49,6 +49,8 @@ pub(crate) fn create(_params: Arc<ViziaPlugParams>) -> Option<Box<dyn Editor>> {
 }
 ```
 
+*Make sure to change instances of `ViziaPlug` to the name of your plugin.*
+
 The `create_vizia_editor` function creates the vizia GUI window with a size determined by the `ViziaState`. For now this state is hard-coded but later we'll move it to the plugin parameters so that the window state is remembered between successive openings of the window.
 
 Within the closure passed to the `create_vizia_editor` function is where we create the vizia GUI and add some controls. In this case we've added a simple label to begin with.

